### PR TITLE
Collapse on all

### PR DESCRIPTION
**Pull request by horsebuzz**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests and did you add any new tests needed for your feature?
2. [x] Did you update all templates (if applicable)?
3. [x] Did you add the [relevant documentation](https://github.com/rappasoft/laravel-livewire-tables-docs) (if applicable)?
4. [x] Did you test locally to make sure your feature works as intended?

### Changes to Core Features:
This PR adds a third option `collapseOnAll`, which will collapse applicable column on all screen sizes. This is an additional feature and other 2 option will works as usual.

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Comment by michel2103** [comment]
Is this functionality working?

File: resources/views/components/table/th/row-contents.blade.php [hunk 1]
```diff
@@ -25,13 +27,14 @@
...
+                        'd-md-none' => !$hasCollapseOnAllColumn && (
                             (($component->shouldCollapseOnMobile() && $component->shouldCollapseOnTablet()) ||
-                            ($component->shouldCollapseOnTablet() && ! $component->shouldCollapseOnMobile()))
+                            ($component->shouldCollapseOnTablet() && ! $component->shouldCollapseOnMobile())))
                     ])
-                    ->class(['d-sm-none' => $component->shouldCollapseOnMobile() && ! $component->shouldCollapseOnTablet()])
+                    ->class(['d-sm-none' => !$hasCollapseOnAllColumn && ($component->shouldCollapseOnMobile() && ! $component->shouldCollapseOnTablet())])
             }}
-        ></th>
+        >Test</th>
```
Comment (GermanZun) [inline resources/views/components/table/th/row-contents.blade.php:38]: is this intended?

Comment (horsebuzz) [reply resources/views/components/table/th/row-contents.blade.php:38]: no typo. I have removed it but not sure how to re-submit pull request

**Comment by horsebuzz** [comment]
function is working, I am using it in my project.

I was hoping that it's merged with master so I do not have to keep track of changes and modify with every release.

**Comment by horsebuzz** [comment]
@GermanZun submitted a new PR request #1047

**Pull request closed without merging**